ParisBNF/abb/BNFabb39.xml

```diff
@@ -402,9 +402,9 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
                                   <note>According to
                                     <bibl><ptr target="bm:Chaine1912Catalogue"/>
                                     <citedRange unit="page">28</citedRange></bibl> the text is an exceprt from Josippon,
-                                    indeed it appears to correspond to the story of Metitan, son of Judah after the edition <bibl><ptr target="bm:Murad1938ZenaAyhud"/>
-                                    <citedRange unit="page">63</citedRange></bibl>,
-                                    but both are similar only in the beginning. The continuation seems to be only a summary based on <ref type="work" corresp="LIT2598Yosipp">Josipppon</ref>
+                                    it appears to correspond to the story of Metitan, son of Judah after the edition <bibl><ptr target="bm:Murad1938ZenaAyhud"/>
```

Comment:
Probably you should add: "At first sight, it seems to correspond ..." - just to make your point clearer and to distance yourself from it, if you feel it appropriate.

Reply:
ok I modified a little. Thanks!